src/shipit_static_analysis/shipit_static_analysis/config.py

```diff
@@ -23,7 +23,7 @@ def __init__(self):
     def setup(self, app_channel):
         self.app_channel = app_channel
         self.download({
-            'cpp_extensions': ['.cpp', '.c', '.h'],
+            'cpp_extensions': ('.c', '.h', '.cpp', '.cc', '.cxx', '.hh', '.hpp', '.hxx', '.m', '.mm'),
```

Comment:
Nit: a `set` could be better here, since you're going to check if elements belong to this

Reply:
Or even better a `frozenset` since it's not going to be modified.